api/src/main/java/keywhiz/api/model/SanitizedSecret.java

```diff
@@ -197,4 +199,30 @@ public static SanitizedSecret fromSecret(Secret secret) {
   public static String displayName(SanitizedSecret sanitizedSecret) {
     return sanitizedSecret.name();
   }
+
+  public static Builder builder() { return new AutoValue_SanitizedSecret.Builder();}
+
+  public abstract Builder toBuilder();
+
+  @AutoValue.Builder
+  public static abstract class Builder {
```

Comment:
I didn't end up using this functionality, but I've been adding builder support to the model classes one by one as I touch them because 15+ argument constructors are ridiculous.